src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Windows.cs

```diff
@@ -5,12 +5,38 @@
 using System.Buffers;
 using System.Diagnostics;
 using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
 
 namespace System.Globalization
 {
     public partial class CompareInfo
     {
-        private unsafe void InitSort(CultureInfo culture)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe IntPtr GetSortHandle(string cultureName)
+        {
+            const uint LCMAP_SORTHANDLE = 0x20000000;
+            const uint LCMAP_HASH = 0x00040000;
```

Comment:
Should these be on Interop.Kernel32 in the same file as LCMapStringEx?

Reply:
I'll move them. I thought about that when changing the code but was lazy to do it :-)